<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data Integrity</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>120</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Serial Settings</string>
    </property>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>127</y>
       <width>103</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Connection Status</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>98</y>
       <width>103</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboParity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>72</y>
       <width>103</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBaud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>46</y>
       <width>103</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>103</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdData0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdData10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="chartView" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Timeout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetZoom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Zoom</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
